# executes a certain motion on the nao

string motion
---
bool success
